### ISSUE TITLE
PEP 541 Request: `pipeline`

### ISSUE THREAD

**Issue by carpetmaker** [pypa/pypi-support#2250]
### Project to be claimed

`pipeline`: https://pypi.org/project/pipeline

### Your PyPI username

`carpetmaker`: https://pypi.org/user/carpetmaker

### Reasons for the request

All criteria for the [Removal of an abandoned project](https://www.python.org/dev/peps/pep-0541/#removal-of-an-abandoned-project) have been met:
+ [x] the project has been determined [abandoned](https://www.python.org/dev/peps/pep-0541/#abandoned-projects)
  + [x] the owner is not [reachable](https://www.python.org/dev/peps/pep-0541/#reachability)
    + [x] via the e-mail address on file in the user's profile on the Package Index
    + [x] via the e-mail address listed in the Author field for a given project uploaded to the Index
    + [x] via any of the e-mail addresses found in the given project's documentation on the Index or on the listed Home Page
  + [x] no releases within the past twelve months
    + last (and only) release: v0.1 (2010-01-17 23:02:31 UTC)
      + project contains a single Python source file from 2007
    + an inactive release (with a lower-than-most-recent version) v0.0.3 (2020-09-22 23:41:27 UTC)
  + [x] no activity from the owner on the project's home page (or no home page listed)
    + [project homepage](https://sourceforge.net/projects/pythoric/) is inactive and also indicates that the current project does not meet notability requirements (project's actual name on sourceforge is different from the one currently being "name-squatted")
+ [x] the candidate is able to demonstrate their own failed attempts to contact the existing owner
  + the author has been contacted via email listed in the pypi project author / maintainer info
([ 📧 Nathan Coulter](mailto:org.python.pypi.pipeline@pooryorick.com)) (no reply whatsoever)
+ [x] the candidate is able to demonstrate that the project suggested to reuse the name already exists and meets notability requirements
    + _Pipeline is a python library used for collection pipelines programming, which drastically simplifies writing of concurrent code
The library is [publicly available](https://github.com/carpetmaker/pipeline) and is fully documented and covered with tests & examples_
+ [x] the candidate is able to demonstrate why a fork under a different name is not an acceptable workaround
    + I authored the library and coincidentally named it Pipeline - the name is not arbitrary, it's rather defined by the existing computer science terminology used in this particular area; the library's component hierarchy also dictates its name - therefore, a fork under a different name is not an acceptable workaround
+ [x] download statistics on the Package Index for the existing package indicate project is not being used
    + statistics indeed [prove](https://libraries.io/pypi/pipeline) that the project is inactive, not used / downloaded at all and also not used as dependency for any packages
+ [ ] the maintainers of the Package Index don't have any additional reservations
  + this is the only pending step required (which is up to PyPI admins / moderators to evaluate & perform)

### Maintenance or replacement?

Replacement

### Source code repositories URLs

+ [Abandoned project source code](https://sourceforge.net/projects/pythoric/files/)
+ [Replacement project source code](https://github.com/carpetmaker/pipeline)

### Contact and additional research

+ Attempts to contact the current author / maintainer have been performed
  + the author has been contacted via email listed in the pypi project author / maintainer info
([ 📧 Nathan Coulter](mailto:org.python.pypi.pipeline@pooryorick.com)) (no reply whatsoever)
  + the author was contacted via [alternative email](mailto:pipeline.py@pooryorick.com) provided in the project source

### Code of Conduct

- [x] I agree to follow the PSF Code of Conduct

**Comment by carpetmaker** [comment]
Dear @yeraydiazdiaz , @ewdurbin & @di !

I am pinging you guys, since you seem to be resolving `PEP 541` issues.
I apologize in advance for the direct ping, however is there a chance any of you three could initiate a _reachability process_?
I sincerely appreciate the amount of volunteering effort you guys put into PSF & PyPA work.
I am aware that the `PEP 541` resolving in its current form is heavily undercapacitated - the backlog is growing faster than the admins can handle the issues - judging by the current number of open issues, I can expect my request to be processed in roughly 6-8 months 😕 .
However, I truly believe all the hard work should be compensated.
That is why I'm willing to make a donation if necessary, to cover those few hours of effort needed to resolve this issue.
Charging a US-python-dev-rate won't be a problem. Just let me know, so we can discuss such details, if needed.
Unfortunately, this issue is a huge blocker for me, and I cannot make my first major FOSS contribution, until I'm able to publish the project under the right sound name to PyPI.

TLDR; I will make the work easier for you, that is why I prepared the [e-mail template](mailto:org.python.pypi.pipeline@pooryorick.com&subject=PEP%20541%20reachability%20process%20-%20PyPI%20package%20%22pipeline%22&body=Dear%20PyPI%20user%2C%0A%0AI%20am%20contacting%20you%20as%20a%20PyPA%20admin%2C%20regarding%20a%20PEP%20541%20request.%0AOne%20of%20the%20users%20initiated%20a%20request%20for%20a%20replacement%20of%20the%20abandoned%20%22pipeline%22%20project.%0ASince%20you%20are%20the%20current%20maintainer%20of%20the%20project%2C%20we%20are%20obliged%20to%20inform%20you%20about%20the%20request%2C%0Aand%20give%20you%20a%206-week%20time%20window%20for%20a%20response%2C%20according%20to%20the%20%22PEP%20541%20-%20reachability%22%20section.%0ASee%3A%20https%3A%2F%2Fpeps.python.org%2Fpep-0541%2F%23reachability%0AIf%20you%20wish%20to%20continue%20maintaining%20the%20project%2C%20you%20need%20to%20reply%20to%20this%20e-mail%20within%20a%20mentioned%206-week%20waiting%20period.%0AKeep%20in%20mind%20that%20by%20doing%20so%2C%20you%20are%20required%20to%20make%20a%20release%20at%20least%20once%20every%2012%20months%2C%20according%20to%20the%20%22PEP%20541%20-%20Abandoned%20projects%22%20section.%0ASee%3A%20https%3A%2F%2Fpeps.python.org%2Fpep-0541%2F%23abandoned-projects%0A%0ABest%20regards%2C%0A%3Cadmin%20name%3E%0APyPA%20Team%20%2F%20PSF), which targets the [currently-abandoned official project maintainer](mailto:org.python.pypi.pipeline@pooryorick.com) listed [here](https://pypi.org/project/pipeline/). If it is not opening, try expanding the section below to see the proposed e-mail template.
<details>
<summary>
Example e-mail template:
</summary>
<br/>

Recipient:
```
org.python.pypi.pipeline@pooryorick.com
```
Subject:
```
PEP 541 reachability process - PyPI package "pipeline"
```
Body:
```
Dear PyPI user,

I am contacting you as a PyPA admin, regarding a PEP 541 request.
One of the users initiated a request for a replacement of the abandoned "pipeline" project.
Since you are the current maintainer of the project, we are obliged to inform you about the request,
and give you a 6-week time window for a response, according to the "PEP 541 - reachability" section.
See: https://peps.python.org/pep-0541/#reachability
If you wish to continue maintaining the project, you need to reply to this e-mail within a mentioned 6-week waiting period.
Keep in mind that by doing so, you are required to make a release at least once every 12 months, according to the "PEP 541 - Abandoned projects" section.
See: https://peps.python.org/pep-0541/#abandoned-projects

Best regards,
<admin name>
PyPA Team / PSF
```
</details>

You can just click on the e-mail, evaluate it before sending it, and voila, that's it. 5 minutes of work to initiate a _reachability process_. But feel free to charge it as if it was few hours of work.

I am looking forward to hearing your reply. Thank you in advance! Once more, srry for the direct ping.
Best regards,
@carpetmaker

**Comment by yeraydiazdiaz** [comment]
Hi @carpetmaker, I will now start the reachability process for your request which can take up to 6 weeks. Hopefully we will hear back from the owner soon and proceed with the transfer.

**Comment by carpetmaker** [comment]
Hello @yeraydiazdiaz !
Thank you for your efforts!
Any updates on this? 6 weeks waiting period has now passed...

**Comment by yeraydiazdiaz** [comment]
I was unable to contact the owner, the transfer is now complete.

**Comment by carpetmaker** [comment]
Thank you for your effort and a prompt response.
